examples/torque_driven_ocp/double_pendulum_WithTranslations.bioMod

```diff
@@ -0,0 +1,104 @@
+version 4
```

Comment:
Please use consistent casing for names (snake case is preferred: double_pendulum_with_translation.bioMod)